docs/LegacyCRD.md

```diff
@@ -0,0 +1,14 @@
+## Utilizing v1beta1 CustomResourceDefinition versions
+The operator supports Kubernetes clusters from v1.13+.
```

Comment:
We support K8s 1.18 and above now
we don't to support legacy CRD no more

Reply:
fixed